zrml/swaps/src/benchmarks.rs

```diff
@@ -365,21 +365,21 @@ benchmarks! {
         let caller: T::AccountId = whitelisted_caller();
         let (pool_id, assets, ..) = bench_create_pool::<T>(caller.clone(), Some(a as usize),
             Some(T::MinLiquidity::get() * 2u32.into()), ScoringRule::CPMM, false);
-        let max_amount_asset_in: BalanceOf<T> = T::MinLiquidity::get();
+        let max_asset_amount_in: Option<BalanceOf<T>> = Some(T::MinLiquidity::get());
         let asset_amount_out: BalanceOf<T> = BASE.saturated_into();
-        let max_price = T::MinLiquidity::get() * 2u32.into();
-    }: swap_exact_amount_out(RawOrigin::Signed(caller), pool_id, assets[0], max_amount_asset_in,
+        let max_price = Some(T::MinLiquidity::get() * 2u32.into());
+    }: swap_exact_amount_out(RawOrigin::Signed(caller), pool_id, assets[0], max_asset_amount_in,
             assets[T::MaxAssets::get() as usize - 1], asset_amount_out, max_price)
 
     swap_exact_amount_out_rikiddo {
         let a in 3 .. T::MaxAssets::get().into();
         let caller: T::AccountId = whitelisted_caller();
         let (pool_id, assets, ..) = bench_create_pool::<T>(caller.clone(), Some(a as usize),
             Some(BASE.saturated_into()), ScoringRule::RikiddoSigmoidFeeMarketEma, true);
-        let max_amount_asset_in: BalanceOf<T> = (BASE * 1024).saturated_into();
+        let max_asset_amount_in: Option<BalanceOf<T>> = Some((BASE * 1024).saturated_into());
         let asset_amount_out: BalanceOf<T> = BASE.saturated_into();
-        let max_price = (BASE * 1024).saturated_into();
-    }: swap_exact_amount_out(RawOrigin::Signed(caller), pool_id, *assets.last().unwrap(), max_amount_asset_in,
+        let max_price = Some((BASE * 1024).saturated_into());
+    }: swap_exact_amount_out(RawOrigin::Signed(caller), pool_id, *assets.last().unwrap(), max_asset_amount_in,
             assets[0], asset_amount_out, max_price)
 }
 
```

Comment:
We could run benchmarks on a per-PR basis. In this case, the benchmarks shouldn't change significantly because we haven't really changed anything. There also seems to be a problem with running benchmarks from a fork, so I think we can skip them for now (and I'm going to stop using a fork for making PRs).